benchmark/src/util.ts

```diff
@@ -0,0 +1,85 @@
+import { BenchmarkFunction } from "./benchmark_types";
+
+export function round(num: number, decimalPlaces: number = 0) {
+    return tonumber(string.format(`%.${decimalPlaces}f`, num));
+}
+
+export const json: {
+    decode: (this: void, str: string) => {};
+    encode: (this: void, val: any) => string;
+} = require("json");
+
+export function readFile(path: string): string | undefined {
+    const fileOpenArray = io.open(path, "rb");
+
+    if (fileOpenArray && fileOpenArray[0]) {
+        const fileHandle = fileOpenArray[0];
+        let fileContent = readAll(fileHandle);
+        fileHandle.close();
+
+        if (fileContent) {
+            return fileContent;
+        }
+    }
+}
+
+export function readAll(file: LuaFile): string | undefined {
+    let content: (string | undefined)[];
```

Comment:
TSLint doesn't appear to be checking these files. We could merge #839 first, or you could change the linting script to include new project

```diff
-"lint:tslint": "tslint -p . && tslint -p test && tslint -p src/lualib",
+"lint:tslint": "tslint -p . && tslint -p test && tslint -p benchmark && tslint -p src/lualib",
```

Reply:
I ran tslint local for /benchmark and fixed all issues but I did not change the config to avoid conflicts with #839.